examples/static_react_task/README.md

```diff
@@ -106,7 +106,7 @@ In order to get started on your own task, it is a good idea to copy this `static
 
 1. Copy the `static_react_task` directory to your project directory
 2. Update `static_task_data` in the `run_task.py` script to provide the required data for your task
-3. Update the task-related `ARG_STRING` variables in `run_task.py` to make sense for your task.
+3. Update anytask-related variables in your `conf/my_new_config.yaml` file to make sense for your task.
```

Comment:
missing a space now 😬... otherwise looks good!

Reply:
This is what I get for a find-replace. Thanks haha

Reply:
Oh wait this is what I get for not saving a file :(